packages/siwe/src/client.ts

```diff
@@ -119,23 +120,35 @@ export class Web3ModalSIWEClient {
         }
       })
     }
-    const signature = await ConnectionController.signMessage(message)
-    const isValid = await this.methods.verifyMessage({ message, signature })
-    if (!isValid) {
-      throw new Error('Error verifying SIWE signature')
-    }
+    try {
+      const signature = await ConnectionController.signMessage(message)
+      const isValid = await this.methods.verifyMessage({ message, signature })
+      if (!isValid) {
+        let errorMessage = 'Error verifying SIWE signature'
+        if (AccountController.state.preferredAccountType === 'smartAccount') {
```

Comment:
I remember we had a constant for this hard coded strings, is it possible to use them here?